### ISSUE TITLE
(MHRise) Crash on startup, "[REFramework] [error] Failed to initialize REFramework on DirectX 12"

### ISSUE THREAD

**Issue by zed213** [praydog/REFramework#236]
My game crashes after downloading only this mod. I took a look at the re2_framework_log and it says this towards the end.

"[REFramework] [error] Failed to initialize REFramework on DirectX 12"

Theres also a reframework_Crash dump file that says this:

Exception Code: 0XC0000005
Exception Information: The thread tried to read from or write to a virtual address for whichh it does not have the apporpriate access.

**Comment by praydog** [comment]
Upload the whole log and dump file.

**Comment by takesword** [comment]
Hey Praydog,

Why are you just reacting to MHRISE?

**Comment by takesword** [comment]
Hey Praydog,


Why are you only answering to MHRISE problems?

**Comment by zed213** [comment]
> Upload the whole log and dump file.
[re2_framework_log.txt](https://github.com/praydog/REFramework/files/7938239/re2_framework_log.txt)
https://www.mediafire.com/file/x2pp0kn4226fyan/reframework_crash.dmp/file

sorry about putting the dump file on mediafire, seems like i cant drag and drop that one on github

**Comment by praydog** [comment]
Are you using something like ReShade or DXVK? Those won't work. What about Proton on Linux?

**Comment by zed213** [comment]
> Are you using something like ReShade or DXVK? DXVK won't work. What about Proton on Linux?

Im on windows 10 and im not using ReShade and I dont know what is DXVK is.

**Comment by praydog** [comment]
Only thing I can recommend right now is try disabling any overlays to try to rule them out as an issue. You're also using a *really* old version of Windows 10. I have not tested if these older versions have compatibility issues with the way we hook DirectX 12.

**Comment by SlitheringSOUL** [comment]
Brought out my old account just to help. . . I solved this be getting rid of the old rise tweaks and updating that bad boy. hope it helps you too. Also, thank you for your hard work praydog. <3

**Comment by zed213** [comment]
> Brought out my old account just to help. . . I solved this be getting rid of the old rise tweaks and updating that bad boy. hope it helps you too. Also, thank you for your hard work praydog. <3

what do you mean by old rise tweaks?

**Comment by SlitheringSOUL** [comment]
(RiseTweaks is a Mod that allows unlocking of fps and resolution for cutscenes) d3d11on12.dll was causing issues for me on the reframework update. the oldest version of a RiseTweaks mod Gave me the same error as you had, After removing it and updating to the script it was solved. if this isn't the case for you, I would disable anything affecting the games engine, whether it be the natural nvidia/amd bs. or maybe even update your directx. also try picking up the latest version of the risetweaks mod because why not. see if anything changes. have you tried the dev build linked in the update release posts here? those are all various things that I tried to solve it. and despite it being mostly basic, I'll put it here anyways. Check your antivirus, I don't have the real explanation for it, but sometimes it gets rid of things or blocks things. high hopes for ya.
